tests/parser/cellml_1_0.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#include "test_utils.h"
+
+#include "gtest/gtest.h"
+
+#include <libcellml>
+
+TEST(ParserTransform, emptyCellml10)
+{
+    const std::string e =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+        "<model xmlns=\"http://www.cellml.org/cellml/1.0#\"/>\n";
+
+    libcellml::ParserPtr parser = libcellml::Parser::create(false);
+    libcellml::ModelPtr model = parser->parseModel(e);
+    EXPECT_EQ(size_t(1), parser->issueCount());
+    EXPECT_EQ(libcellml::Issue::Level::MESSAGE, parser->issue(0)->level());
+    EXPECT_EQ("Given model is a CellML 1.0 model, the parser will try to represent this model in CellML 2.0.",
+              parser->issue(0)->description());
+}
+
+TEST(ParserTransform, attemptToParseUnknownXmlModelWithPermissiveParser)
+{
+    const std::string n = "name";
+    const std::string e =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+        "<model xmlns=\"http://www.cellml.org/cellml/1.3#\" name=\"name\"/>\n";
+
+    libcellml::ParserPtr parser = libcellml::Parser::create(false);
+    libcellml::ModelPtr model = parser->parseModel(e);
+
+    EXPECT_EQ("", model->name());
+    EXPECT_EQ(size_t(1), parser->issueCount());
+    EXPECT_EQ(libcellml::Issue::Level::ERROR, parser->issue(0)->level());
+    EXPECT_EQ("Model element is in an invalid namespace 'http://www.cellml.org/cellml/1.3#'.", parser->issue(0)->description());
+}
+
+TEST(ParserTransform, attemptToParseUnknownXmlModelWith20Parser)
+{
+    const std::string n = "name";
+    const std::string e =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+        "<model xmlns=\"http://www.cellml.org/cellml/1.3#\" name=\"name\"/>\n";
+
+    libcellml::ParserPtr parser = libcellml::Parser::create();
+    libcellml::ModelPtr model = parser->parseModel(e);
+
+    EXPECT_EQ("", model->name());
+    EXPECT_EQ(size_t(1), parser->issueCount());
+    EXPECT_EQ(libcellml::Issue::Level::ERROR, parser->issue(0)->level());
+    EXPECT_EQ("Model element is in an invalid namespace 'http://www.cellml.org/cellml/1.3#'. A valid CellML root node should be in the namespace 'http://www.cellml.org/cellml/2.0#'.", parser->issue(0)->description());
```

Comment:
Replace `A valid CellML root node...` with `A model element...`, to be consistent with the previous sentence (which might start with `A model element...`?).